scripts/bootstrap-py2-venv

```diff
@@ -0,0 +1,53 @@
+# This script is meant to be sourced by bash, not executed.
+# As such, don't use exit, use return.
```

Comment:
You can verify that it's being sourced

example: https://github.com/EvanPurkhiser/dots-personal/blob/master/init#L8